pkg/controller/reconciler/argocd_reconciler.go

```diff
@@ -78,14 +78,26 @@ func (z *argocdReconciler) Reconcile(node *resourcetree.ResourceNode) (*Reconcil
 			AuthDomain:         resourceState.AuthDomain,
 			Repository:         repository,
 		})
+
 		if err != nil {
-			return &ReconcilationResult{Requeue: true}, fmt.Errorf("error creating argocd: %w", err)
+			// nolint: godox
+			// TODO: Need to identify the correct error
+			if strings.Contains(strings.ToLower(err.Error()), "timeout") {
+				fmt.Println(z.commonMetadata.Out, fmt.Errorf("got ArgoCD timeout: %w", err).Error())
```

Comment:
Should this be wrapped in a `if debug {}`?

Reply:
Could be, but without it we might get the correct error we want to check on later from a user. IDUNNO. But I'm off work and it would be nice to get this merged by tonights nightly :P